pom.xml

```diff
@@ -221,54 +461,54 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>com.mycila</groupId>
-        <artifactId>license-maven-plugin</artifactId>
-        <version>${license.maven.plugin.version}</version>
-        <dependencies>
-          <dependency>
-            <groupId>com.mycila</groupId>
-            <artifactId>license-maven-plugin-git</artifactId>
-            <version>${license.maven.plugin.version}</version>
-          </dependency>
-        </dependencies>
-        <configuration>
-          <header>src/main/license/APACHE-2.txt</header>
-          <properties>
-            <owner>The HiveRunner Contributors</owner>
-          </properties>
-          <includes>
-            <include>src/main/java/**</include>
-            <include>src/test/java/**</include>
-          </includes>
-        </configuration>
-        <executions>
-          <execution>
-            <phase>validate</phase>
-            <goals>
-              <goal>format</goal>
-            </goals>
-          </execution>
-        </executions>
-      </plugin>
-      
+<!--      <plugin>-->
```

Comment:
I'll need to look into this, it should "just work" but I remember your previous set of changes it seemed to be doing something weird.

Reply:
sure, thanks